modules/serverless-tech-preview-features.adoc

```diff
@@ -110,5 +110,15 @@ The following table provides information about which {ServerlessProductName} fea
 |GA
 |GA
 
+|Pipelines-as-code
```

Comment:
In both TP/GA and Deprecated tables, we add new entries from the top, not from the bottom. So, move this to the top, too.

Reply:
Added the list on the top